### PR TITLE
Added contributing codeowners and checklist files

### PR DESCRIPTION
**Pull request by asa1997**

### CODE REVIEW

File: CONTRIBUTING
Comment (harimohanr) [inline CONTRIBUTING:1]: @asa1997 The .md extension is missing for this file.

File: CONTRIBUTING [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# Contributing to BeSman Environment Repo
+
+Thank you for contributing your time and expertise to the BeSman Environment Repo. This document provides guidance on environments contribution recommended practices. It covers what we're looking for in order to help set expectations and help you get the most out of participation in this project.
+
+This repo contains scripts which BeSman uses to set-up environments for projects/ml models...The repo is maintained by Be-Secure Community and the owners of BeSman.
+Eventhough this is a separate repo, it has a close coupling with BeSman. All the bugs and enhancements for the environment repo will be maintained in the [issue](https://github.com/Be-Secure/BeSman/issues) section of BeSman.
```
Comment (harimohanr) [inline CONTRIBUTING:6]: @asa1997 Why would the environments specific issues be maintained in BeSman issues? Is this a copy and paste mistake?

File: CONTRIBUTING [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+# Code of Conduct
+
+This project adheres to the Contributor Covenant code of [conduct](https://github.com/Be-Secure/BeSman/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
```
Comment (harimohanr) [inline CONTRIBUTING:10]: @asa1997 Have a Code of Conduct file for this repo as well.

File: CONTRIBUTING [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+# Using GitHub Issues
+
+We use GitHub [issues](https://github.com/Be-Secure/BeSman/issues) BeSman to track bugs and enhancements. If you have a general question, you can start a discussion [here](https://github.com/Be-Secure/BeSman/discussions).
```
Comment (harimohanr) [inline CONTRIBUTING:14]: @asa1997 BeSman issues URL again..

File: CONTRIBUTING [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+If we request changes, try to make those changes in a timely manner. Otherwise, PRs can go stale and be a lot more work for all of us to merge in the future.
+
+Even with everyone making their best effort to be responsive, it can be time-consuming to get a PR merged. It can be frustrating to deal with the back-and-forth as we make sure that we understand the changes fully. Please bear with us, and please know that we appreciate the time and energy you put into the project.
+
+# PR Checks
+
+The following checks run when a PR is opened:
+
+1. Contributor License Agreement (CLA): If this is your first contribution to BeSman you will be asked to sign the CLA.
```
Comment (harimohanr) [inline CONTRIBUTING:77]: @asa1997 BeSman again..

File: CONTRIBUTING (hunk 2 above)
Comment (asa1997) [reply CONTRIBUTING:10]: @harimohanr Already present

File: CONTRIBUTING
Comment (asa1997) [reply CONTRIBUTING:1]: Done

File: CONTRIBUTING (hunk 4 above)
Comment (asa1997) [reply CONTRIBUTING]: Changed

File: CONTRIBUTING (hunk 3 above)
Comment (asa1997) [reply CONTRIBUTING]: Issues will be raised against BeSman

**Review by harimohanr — APPROVED** [review]
@asa1997 Looks okay

**Pull request merged**